Sources/SwiftFrameCore/Workers/GraphicsContext.swift

```diff
@@ -0,0 +1,37 @@
+import AppKit
+import CoreGraphics
+import Foundation
+
+class GraphicsContext {
+
+    let cg: CGContext
+    private let colorSpace: CGColorSpace
+
+    lazy var ci: CIContext = {
+        CIContext(cgContext: cg, options: [
+            CIContextOption.workingColorSpace: colorSpace,
+            CIContextOption.useSoftwareRenderer: false
+        ])
+    }()
+
+
+    init(canvasSize: CGSize) throws {
```

Comment:
Nit: Just `size` might be enough. There's no "canvas" involved elsewhere in this class from what I see.